css-cascade/css-cascade-quiz-notes.md

```diff
@@ -0,0 +1,49 @@
+# css-cascade-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What are the four components of "the Cascade".
+  1 user-agent (browser)
+  2 user
+  3 author (developer)
+  4 CSS @keyframe animations
```

Comment:
Not quite! Think about what we talked about in this exercise. What are the names of your files?